Bluepill-cli/BPInstanceTests/BluepillTests.m

```diff
@@ -92,7 +92,7 @@ - (void)testAppThatCrashesOnLaunch {
     self.config.testing_CrashAppOnLaunch = YES;
     self.config.stuckTimeout = @3;
     BPExitStatus exitCode = [[[Bluepill alloc ] initWithConfiguration:self.config] run];
-    XCTAssert(exitCode == BPExitStatusAppCrashed, @"Expected: %ld Got: %ld", (long)BPExitStatusAppCrashed, (long)exitCode);
+    XCTAssert(exitCode == BPExitStatusAppTerminatedBeforeTestStarted, @"Expected: %ld Got: %ld", (long)BPExitStatusAppTerminatedBeforeTestStarted, (long)exitCode);
```

Comment:
I don't get this. If app terminate before test started, it means app crashed.

Reply:
XCTest do terminate app when it timeout establish a control session, which is the bug introduced in Xcode 10

Reply:
then it is not terminated but timed out, we should make it more explicit

Reply:
The app is not time out or crashed by itself. it is terminated by XCTest. And in other cases it could be app crashed by itself. Both are possible. "timeout" does not look more explicit to me.

Reply:
`you said xctest terminate app if timeout establish a control session` this sounds have nothing to do with crash?

Reply:
yes. but there is another possibility for this code get hit, that is app crash by itself before test start as the original log message says

Reply:
If that's the case and we merge them into one unclear exit code, that is more confusing

Reply:
what's your suggestion?

Reply:
`BPExitStatusXCTestConnectionTimeout`